<launch>
    <!-- 启动乌龟模拟器 -->
    <node pkg="turtlesim" type="turtlesim_node" name="turtlesim" output="screen"/>

    <!-- 设置Python节点的参数 -->
    <param name="package_path" type="str" value="$(find turtlr_velocity)"/>

    <!-- 启动Python速度发布节点 -->
    <node pkg="turtlr_velocity" type="turtle_vel_cmd.py" name="turtle_vel_publisher" output="screen"/>

    <!-- 启动C++控制节点 -->
    <node pkg="turtlr_velocity" type="turtleism_control" name="turtle_controller" output="screen"/>
</launch>
